config/cmake/mccacheinit.cmake

```diff
@@ -11,9 +11,9 @@
 #
 # This is the CMakeCache file.
 
-########################
+#########################
 # EXTERNAL cache entries
```

Comment:
mc stands for "memory check".

Reply:
So line 15 should be "# EXTERNAL cache entries for memory checking"?

Reply:
Or something like it.